src/robot/model/keyword.py

```diff
@@ -65,7 +73,7 @@ def to_dict(self):
         return data
 
 
-class Keywords(ItemList):
+class Keywords(ItemList[Keyword]):
```

Comment:
Although this class is called `Keywords`, it actually contains `BodyItems`. Not a huge deal either way, it will be removed in RF 7.